OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -165,7 +165,7 @@ void ConnectionStateChanged(OrderManager om)
 			Func<bool> configurationDisabled = () => !Game.IsHost || gameStarting ||
 				panel == PanelType.Kick || panel == PanelType.ForceStart ||
 				!map.RulesLoaded || map.InvalidCustomRules ||
-				orderManager.LocalClient == null || orderManager.LocalClient.IsReady;
+				orderManager.LocalClient == null || orderManager.LocalClient.IsReady || map.PlayerCount > 64;
```

Comment:
This is already checking `map.InvalidCustomRules`, so we could avoid the special case check here too.